src/test/java/robotbuilder/TestUtils.java

```diff
@@ -177,11 +177,11 @@ public static RobotTree generateTestTree() {
         leftstick.add(armUpButton);
         RobotComponent autoButton = new RobotComponent("Autonomous Button", "Joystick Button", tree);
         rightstick.add(autoButton);
-        autoButton.getProperty("When to Run").setValueAndUpdate("whenPressed");
+        autoButton.getProperty("When to Run").setValueAndUpdate("onTrue");
         RobotComponent xbox = new RobotComponent("Xbox", "Xbox Controller", tree);
         oi.add(xbox);
         RobotComponent xboxButton = new RobotComponent("Arm Up Xbox Button", "Xbox Button", tree);
-        xboxButton.getProperty("When to Run").setValueAndUpdate("whenPressed");
+        xboxButton.getProperty("When to Run").setValueAndUpdate("whileTrue");
```

Comment:
This looks like a copypasta mistake

Reply:
that was intentional to get coverage of additional types